gossipd/gossip.c

```diff
@@ -1011,6 +1011,32 @@ static struct io_plan *hand_back_peer(struct io_conn *conn,
 			  read_returning_gossipfd, rpeer);
 }
 
+static struct io_plan *disconnect_peer(struct io_conn *conn, struct daemon *daemon,
+				       const u8 *msg)
+{
+	struct pubkey id;
+ 	struct peer *peer;
+
+	if (!fromwire_gossipctl_peer_disconnect(msg, &id))
+		master_badmsg(WIRE_GOSSIPCTL_PEER_DISCONNECT, msg);
+
+	peer = find_peer(daemon, &id);
+	if (peer && peer->local) {
+		io_close(peer->local->conn);
+		msg = towire_gossipctl_peer_disconnect_reply(msg);
+		daemon_conn_send(&daemon->master, take(msg));
+	} else {
+		status_trace("release_peer: peer %s %s",
```

Comment:
Function is named `disconnect_peer`...

Reply:
I will rename the disconnect handler function to `peer_disconnect`

Reply:
`peer_disconnect` sounds more like "the peer disconnects from me", while `disconnect_peer` sounds more like "I disconnect from the peer", so I think `disconnect_peer` for this function is good.

Reply:
What I am commenting on here is that the status message starts with `release_peer:` when the function that prints the message is named `disconnect_peer`.

Reply:
ahh yes. That makes sense. Will change.